### PR TITLE
Extension Pricing Label docs

### PR DESCRIPTION
**Pull request by prashantvc**
Added the documentation to provide extension pricing label

### CODE REVIEW

File: api/working-with-extensions/publishing-extension.md [hunk 1]
```diff
@@ -239,6 +239,24 @@ Eligible domains meet the following criteria:
 - Your domain must use HTTPS protocol.
 - Your domain must be able to serve an HTTP 200 status response to a HEAD request.
 
+### Extension Pricing Label
+
+You can add a pricing label to your extension in the Marketplace to indicate that it is `FREE` or `FREE TRIAL`.
+
+As an extension author you can opt-in to dispaly a pricing label on your extension by adding the `pricing` field to your `package.json`, example:
```
Comment (ProKashif) [inline api/working-with-extensions/publishing-extension.md:246]: typo 

```suggestion
As an extension author, you can opt-in to display a pricing label on your extension by adding the `pricing` field to your `package.json`, example:
```

File: api/working-with-extensions/publishing-extension.md [hunk 2]
```diff
@@ -239,6 +239,24 @@ Eligible domains meet the following criteria:
...
+{
+  "pricing": "Free"
+}
+```
+
+The value of the `pricing` field can be either `Free` or `Trial`. Note that these values are case-sensitive.
+
+If you do not add the `pricing` field to your `package.json`, the default label will be `FREE`.
+
+>**Note:** Make sure to use the `vsce` version >= `2.10.0` when publishing your extension for pricing label to work.
```
Comment (ProKashif) [inline api/working-with-extensions/publishing-extension.md:258]:
```suggestion
>**Note:** Make sure to use the `vsce` version >= `2.10.0` when publishing your extension for the pricing label to work.
```

**Review by ProKashif — APPROVED** [review]

**Review by fred231084g — APPROVED** [review]

File: api/working-with-extensions/publishing-extension.md (hunk 1 above)
Comment (sa5544) [reply api/working-with-extensions/publishing-extension.md:246]: Reple...

Comment (sa5544) [reply api/working-with-extensions/publishing-extension.md:246]: Reply...

**Comment by prashantvc** [comment]
Can we merge this please?

**Comment by gregvanl** [comment]
Hi @prashantvc Are you missing a commit to fix the couple of resolved typos?

**Comment by gregvanl** [comment]
Looks like @prashantvc may be done for the day. I'll merge this PR and fix the typos in a follow up PR.

**Pull request merged**